meilisearch-http/tests/dump.rs

```diff
@@ -367,20 +369,10 @@ async fn dump_index_updates_should_be_valid() {
     compression::from_tar_gz(&dumps_dir.join(&format!("{}.dump", uid)), tmp_dir_path).unwrap();
 
     let file = File::open(tmp_dir_path.join("test").join("updates.jsonl")).unwrap();
-    let mut updates = read_all_jsonline(file);
-
-
-    // hotfix until #943 is fixed (https://github.com/meilisearch/MeiliSearch/issues/943)
-    updates.as_array_mut().unwrap()
-            .get_mut(0).unwrap()
-            .get_mut("type").unwrap()
-            .get_mut("settings").unwrap()
-            .get_mut("displayed_attributes").unwrap()
-            .get_mut("Update").unwrap()
-            .as_array_mut().unwrap().sort_by(|a, b| a.as_str().cmp(&b.as_str()));
```

Comment:
Do I need to understand that you have also fixed #943?

Reply:
no, this is related to the fact that there initially was an update on server creation where `searchableAttributes` and `displayedAttributes` were specified. see my comment above.

This code simply reordered the attributes to make the test happy, but since these fields are  not there anymore, it made the test break.